packages/@okta/vuepress-site/docs/guides/terraform-manage-end-user-experience/main/index.md

```diff
@@ -1,463 +1,662 @@
+              </td>
+          </tr>
+      </table>
+    </td></tr>
+    <tr>
+      <td>
+          <table style="width: 100%; line-height: 20px; padding: 32px;" cellpadding="0">
+              <tr>
+                  <td style="font-size: 12px; padding-top: 24px; color: #999;">
+                      This email was automatically sent using <a href="https://www.okta.com" style="color:#616161">Okta's service</a>. Replies are not monitored or answered. Okta has can't see who receives these emails, and is not responsible for, and disclaims any and all liability associated with, this email's content. <br/><br/>The sender may have customized the contents of the email. If you would like to notify Okta of anything suspicious, report it to security@okta.com.
+                  </td>
+              </tr>
+          </table>
+      </td>
+    </tr>
+  </table>
+  </div>
+  </body>
+  </html>
+  HTMLDELIMITER
+}
+```
+
+The following image shows an example of an email generated using the template:
+
+<div class="three-quarter border">
+
+![An activation email with personal information and URLs hidden by gray rectangles.](/img/terraform/activate-email.png)
+
+</div>
 
-## Customize the appearance of your org
+## Create a custom email domain
 
-Change the colors, logo, and background image of your sign-in page, Okta End User Dashboard, and error pages. Use the `okta_brand` and `okta_theme` resources that you imported into your Terraform configuration.
+Use a custom domain for emails sent from Okta instead of the default email address of `noreply@okta.com`.
 
-To customize the appearance of your org:
+Okta polls your custom domain once daily to confirm its operational status. If the poll fails, Okta alerts the super administrators by email. It also starts using `noreply@okta.com` until the issue is resolved.
 
-1. Modify the argument values in the `okta_brand` resource.
+There are two limitations to email domain customization:
 
-   * Set `agree_to_custom_privacy_policy` to `true` and `custom_privacy_policy_url` to use your own privacy policy in the footer of your Okta pages. This replaces the default private policy link.
-   * Set `remove_powered_by_okta` to `true` to remove “Powered by Okta” from the sign-in page.
+* Okta can't send an email from a domain that uses [SendGrid.](https://sendgrid.com/) You can configure a subdomain with your DNS provider for custom Okta emails.
+
+* There's a maximum of 10 DNS lookups in an SPF record.
+
+Okta recommends that your domain implement the [Sender Policy Framework (SPF) ](https://tools.ietf.org/html/rfc7208)to prevent sender address forgery. If you have already implemented SPF in your custom domain, update the SPF record to include Okta's mail servers.
```

Comment:
```suggestion
Okta recommends that your domain implement the [Sender Policy Framework (SPF)](https://tools.ietf.org/html/rfc7208) to prevent sender address forgery. If you have already implemented SPF in your custom domain, update the SPF record to include Okta mail servers.
```

Reply:
Fix is in next commit.